pkg/operator/controller/controller_test.go

```diff
@@ -1523,12 +1523,14 @@ func createArgs() *args {
 }
 
 func doReconcile(args *args) {
-	result, err := args.reconciler.Reconcile(context.TODO(), reconcileRequest(args.cdi.Name))
-	Expect(err).ToNot(HaveOccurred())
-	Expect(result.Requeue).To(BeFalse())
+	Eventually(func() bool {
+		result, err := args.reconciler.Reconcile(context.TODO(), reconcileRequest(args.cdi.Name))
+		return err == nil && !result.Requeue
+	}, 1*time.Minute, 2*time.Second).Should(BeTrue())
```

Comment:
What's going on here?  Unit tests should be deterministic.

Reply:
I think I get it

Reply:
Thanks. Reverted it.